### ISSUE TITLE
VideoLanguage Resource Updates

### ISSUE THREAD

**Issue by bendk** [pculture/unisubs#2008]
There a few fields in the video language resource that should probably change:

* Drop `is_original` and use `is_primary_audio` instead
* Drop `num_versions` -- clients can just get the length of the versions array.
* Drop `id` field -- clients don't need to know our internal ids, and it's not used in the API anyway.
* Drop `original_language_code` and `is_translation` -- We're not tracking translations anymore with the new data model

**Comment by franontanaya** [comment]
> Drop num_versions -- clients can just get the length of the versions array.

I agree a lot with this one. num_versions is not zero-based and the array length usually would be, which is an invitation for off-by-one errors.

> Drop id field -- clients don't need to know our internal ids, and it's not used in the API anyway.

The ID can be used to craft links to the language's page, but "site_url" already provides this -- so it's ok to remove the language ID because it's redundant, but this wouldn't hide the language ID if that was the goal. I use the language id/site_url myself to add the language's link to spreadsheets and to check the sync tab.

**Comment by bendk** [comment]
Good point on the language ID.  I personally don't understand why we include it on the site pages and think it could be removed there too.  I don't see us supporting multiple subtitle sets for the same language code anytime soon.  That said, I don't think we should remove it from the API data while it's still used in the URLs.
